### PR TITLE
testmap: change default test OS for cockpit-composer to fedora-34

### PR DESCRIPTION
**Pull request by KKoukiou**
*  [ ] blocked osbuild/osbuild-composer#1285

### CODE REVIEW

File: lib/testmap.py [hunk 1]
```diff
@@ -133,8 +133,8 @@
     },
     'osbuild/cockpit-composer': {
         'master': [
-            'fedora-33',
-            'fedora-33/firefox',
+            'fedora-34',
+            'fedora-34/firefox',
```
Comment (marusak) [inline lib/testmap.py:137]: Will you be able to trigger this one or wanna help with that?

**Comment by martinpitt** [comment]
Post "http://localhost/api/v1/blueprints/new": read unix @->/run/weldr/api.socket: read: connection reset by peer

Some regression in osbuild on F34? @teg @jkozol ?

**Comment by jkozol** [comment]
osbuild-composer does not currently support fedora-34 so cockpit-composer cannot run on fedora-34. This PR should be blocked until https://github.com/osbuild/osbuild-composer/pull/1285 merges.

**Comment by martinpitt** [comment]
This has been stalled for too long, and keeps old images on our servers. Closing for now. We can reopen once cockpit-composer gets fixed.

**Pull request closed without merging**